packages/data/json/9.10.14.json

```diff
@@ -3,10 +3,10 @@
     "title": "Spatial Separation Between Buildings",
     "subtitle": "Volume II - 9.10.14.1-4",
     "description": "Ensure a building’s separation passes code.",
-    "startingSectionId": "intro"
+    "startingSectionId": "1_intro"
```

Comment:
Why are we adding a 1_ and 2_ in front of the sectionIds?

Reply:
This is to take advantage of the ordering of the sections. When using Object.keys to turn the object of sections into an array to iterate over for display in order, I wanted them to be in a specific order. "number-like" keys use ascending order.